.prettierignore

```diff
@@ -0,0 +1 @@
+hardhat.config.ts
```

Comment:
Why?

Reply:
> I don't like the fact that we would have to take care of the order (first format, then lint). You can freely introduce the mentioned plugin if it resolves this issue.

The plugin mentioned in the old PR description doesn't offer a CLI out of the box, [this](https://github.com/Shard-Labs/starknet-hardhat-example/pull/74/commits/14dca5d1bc47a34696a306f36f335d59a0a50921) introduce a new one with the same functionality.